src/sentry/tasks/symbolication.py

```diff
@@ -163,11 +168,11 @@ def _continue_to_process_event(was_killswitched: bool = False) -> None:
         # After JS processing, we check `get_native_symbolication_function`,
         # because maybe we need to feed it to another round of
         # `symbolicate_event`, but for *native* that time.
-        if not was_killswitched and task_kind.is_js:
+        if not was_killswitched and task_kind.platform == SymbolicatorPlatform.js:
```

Comment:
I don’t see any code handling the case the above comment mentions.

The problem here is that all the different tasks delegate to the same internal function.
As well as the fact that we might now need to pass it *three* times through the same function, one after the other, as in theory we could have a mixed stack trace involving *all* the languages.

That BTW is not really "in theory" as React Native on Android does indeed involve all three platforms.

The logic that decides which inner symbolication function to use, and to which other symbolication task to forward the event to is really horrible :-(
Ideally, I would love to have some kind of bitfield for this, which has markers for which platforms / symbolication functions this needs to run through, vs which ones it has already run through. Together with a function that uses this info to then forward the event to the appropriate next task.